andy_api/api/intent_processing/restart_game.py

```diff
@@ -0,0 +1,18 @@
+"""
+TODO add info about intent
+"""
+
+from .utils import get_random_choice
+from api.state_manager import restart_game
+from api.chess_logic import get_new_board
+
+HAPPY_PATH_RESPONSES = [
+    "Okay- let's play again!",
+    "Lets restart the round..."
+]
```

Comment:
Add follow-up intents:

1. Make fulfillment handler for yes and no
2. Change HAPPY_PATH_RESPONSES to ask the user if they want to restart a game
3. Restart game fulfillment: returns a static response every time
4. Restart game Yes fulfillment: has the logic in it
5. Restart game No fulfillment: returns a static response every time